appveyor.yml

```diff
@@ -2,8 +2,6 @@ version: '{build}'
 os: Visual Studio 2015
 install:
 - set PATH=%ProgramFiles(x86)%\MSBuild\14.0\Bin;%PATH%
-- dnvm update-self
-- dnvm install 1.0.0-rc1-update1
```

Comment:
As far as I know it no longer works that way in RC2. There is no equivalent to dnvm in the new toolset. If you have .NET Core RC2 installed already, you can just get the ASP.NET Core packages using NuGet.
That was one of the biggest changes from RC1 to RC2, and it was done in order to make all .NET Core environments consistent with each other.